package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "parcel src/index.hbs --no-cache --open",
+    "build": "parcel build src/index.hbs --no-cache",
+    "start": "parcel build src/index.hbs --no-cache && node server.js"
```

Можно переиспользовать
```suggestion
    "start": "npm run build && node server.js"
```